src/main/java/no/sikt/nva/scrapers/DublinCoreScraper.java

```diff
@@ -844,6 +844,35 @@ private void setIdFromJournals(BrageLocation brageLocation, Record record) {
         if (isJournal(record)) {
             setChannelRegisterIdentifierForJournal(brageLocation, record);
         }
+        if (isBook(record)) {
+            setChannelRegisterIdentifierForBook(brageLocation, record);
+        }
+    }
+
+    private void setChannelRegisterIdentifierForBook(BrageLocation brageLocation, Record record) {
+        record.getPublication()
+            .getPublicationContext()
+            .setSeries(
+                new Series(extractChannelRegisterIdentifierForSeries(brageLocation, record.getPublication())));
+    }
+
+    private String extractChannelRegisterIdentifierForSeries(BrageLocation brageLocation, Publication publication) {
+        return partOfSeriesIsPresent(publication)
+                   ? channelRegister.lookUpInJournalByTitle(publication.getPartOfSeries().getName(), brageLocation)
+                   : null;
+    }
+
+    private static boolean partOfSeriesIsPresent(Publication publication) {
+        return Optional.ofNullable(publication)
+                   .map(Publication::getPartOfSeries)
+                   .map(PartOfSeries::getName)
+                   .isPresent();
+    }
+
+    private boolean isBook(Record record) {
+        return NvaType.BOOK.getValue().equals(record.getType().getNva())
```

Comment:
Nitpick, but maybe easier to read with the compared value extracted? I.e. something like this:

```
var nva = record.getType().getNva();
return NvaType.BOOK.getValue().equals(nva)
    || NvaType.TEXTBOOK.getValue().equals(nva)
    || NvaType.BOOK_OF_ABSTRACTS.getValue().equals(nva);
```